polyfill/test/validStrings.mjs

```diff
@@ -293,20 +293,11 @@ const timeZoneNumericUTCOffsetNotAmbiguousAllowedNegativeHour = withCode(
   saveOffset
 );
 const timeZoneUTCOffset = choice(utcDesignator, timeZoneNumericUTCOffset);
-const timeZoneUTCOffsetName = seq(
```

Comment:
Doesn't `timeZoneUTCOffsetName` still have to be included in `timeZoneIANAName` to match the grammar in the spec?

Reply:
I guess we can drop it here because it gets imported thanks to `timezones.json`?

Reply:
The offset names such as `-08:00` are not part of `timezones.json`, so they still need to be included separately.

Reply:
ah, OK, I think I understand now; there was some confusion on my end. The use of "name" here made me think that we weren't talking about numbers, but rather natural language names. It seems weird to me to call "-08:00" a name (I'd just call it a number or numeric or numerical offset).

production restored